apps/ae_backend_service/lib/backend_session.ex

```diff
@@ -96,8 +96,10 @@ defmodule BackendSession do
   end
 
   # once this occured we should be able to reconnect.
+  # if we don't update the channel id somewhere here,
+  # channel_id will be known as nil by BackendServiceManager
   def handle_cast({:channels_info, method, channel_id}, state)
-      when method in ["funding_signed", "funding_created"] do
+      when method in ["funding_signed", "funding_created", "open"] do
```

Comment:
```suggestion
      when method in ["funding_signed", "funding_created"] do
```
I would assume that the reason you had to make this changes is that you are not using the ae master branch?